packages/dds/tree/src/simple-tree/api/schemaFactoryAlpha.ts

```diff
@@ -628,23 +632,24 @@ export class SchemaFactoryAlpha<
 	}
 
 	/**
-	 * Define a {@link TreeNodeSchema} for a {@link (TreeArrayNode:interface)}.
+	 * Define a {@link TreeNodeSchema} for a {@link (TreeRecordNode:interface)}.
 	 *
 	 * @param name - Unique identifier for this schema within this factory's scope.
 	 *
 	 * @remarks
-	 * This is not intended to be used directly, use the overload of `array` which takes a name instead.
-	 * This is only public to work around a compiler limitation.
+	 * This is not intended to be used directly, use the overload of `record` which takes a name instead.
```

Comment:
Given that this is private, do we need this remarks block at all?

Reply:
I don't think it's necessary, but I also don't think its harmful.